funcr/funcr_test.go

```diff
@@ -738,6 +751,26 @@ func TestRender(t *testing.T) {
 		}{"arg", 789}, "val"),
 		expectKV:   `"<non-string-key: {"F1":"builtin",>"="val" "<non-string-key: {\"F1\":\"value\",\"F>"="val" "<non-string-key: {\"F1\":\"arg\",\"F2\">"="val"`,
 		expectJSON: `{"<non-string-key: {"F1":"builtin",>":"val","<non-string-key: {\"F1\":\"value\",\"F>":"val","<non-string-key: {\"F1\":\"arg\",\"F2\">":"val"}`,
+	}, {
+		name:       "json rendering with json.RawMessage",
+		args:       makeKV("key", raw),
+		expectKV:   `"key"={"message":[123,34,105,110,116,49,34,58,48,44,34,45,34,58,48,44,34,73,110,116,53,34,58,48,125]}`,
+		expectJSON: `{"key":{"message":{"int1":0,"-":0,"Int5":0}}}`,
+	}, {
+		name:       "byte array not json.RawMessage",
+		args:       makeKV("key", []byte{1, 2, 3, 4}),
+		expectKV:   `"key"=[1,2,3,4]`,
+		expectJSON: `{"key":[1,2,3,4]}`,
+	}, {
+		name:       "json rendering with empty json.RawMessage",
+		args:       makeKV("key", &Trawjson{}),
+		expectKV:   `"key"={"message":[]}`,
+		expectJSON: `{"key":{"message":{}}}`,
+	}, {
+		name:       "byte array not json.RawMessage",
```

Comment:
this case seems to be a dup of the above?

Reply:
Yep. Must have cut/paste. Cleaned up